engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/views/register-oracle-client.html

```diff
@@ -0,0 +1,5 @@
+<li>
+    <p translate>Install the subscription manager:</p>
```

Comment:
Lets change this to either:
`Install the Subscription Manager package:` or `Install Subscription Manager`